### DISCUSSION TITLE
Is KEDA a Node Autoscaler  ?

### DISCUSSION THREAD

**Discussion by mandeepgoyat** [kedacore/keda#3050] [Q&A / Need Help]
Hi Team

I am going through KEDA documentation and trying to understand whether I can use it to scale up my k8s nodes as well or not ?
So far I understood KEDA is a pod autoscaler but not a Node Autoscaler like Karpenter or Cluster Autoscaler.

**Comment by tomkerkhove** [comment]
Hi, as the name suggests we purely focus on scaling applications and not nodes.

For that, you'll need to use Cluster Autoscaler or Karpenter indeed.
